customization-guide.md

```diff
@@ -0,0 +1,106 @@
+# IMS Provisioning for Wazi environment
+
+The IMS provisioning playbook samples demonstrate how to allocate the required
+data sets and configure them to provision IMS and related services.
+
+It is a good practice to review the playbook sample contents before executing
+them. It will help you understand the requirements in terms of space, location,
+names, authority, and the artifacts that will be created and cleaned up.
+Although samples are written to operate without the need for the user’s
+configuration, flexibility is written into the samples because it is not easy
+to determine if a sample has access to the host’s resources. Review the
+playbook notes sections for additional details and configuration.
+
+
```

Comment:
Should we add a section for operator customization explaining which operator or collection files need to be updated?

At a quick glance, I think:
- In `galaxy.yml`, the `name`, `namespace`, `version`, `description` and `authors` will need to be modified.
- In `operator-config.yml,` the `name`, `domain`, `version`, `description` and `icon` will need to be updated
- The `zoscb` tag may need to be removed from `galaxy.yml` or else the customized collection can show up in z/OS Cloud Broker's Galaxy Import page (if they upload the customized collection to Galaxy).

If some of these changes aren't made, it can have unintended side effects on the users OpenShift test environment after the customized collection is mapped. For example, the customized operator CRD could override the original operator CRD version on the cluster and affect original operator's instances - we've seen this happen in our internal testing.